aspect/intellij_info_impl.bzl

```diff
@@ -203,7 +203,7 @@ def update_set_in_dict(input_dict, key, other_set):
 
 def collect_py_info(target, ctx, semantics, ide_info, ide_info_file, output_groups):
     """Updates Python-specific output groups, returns false if not a Python target."""
-    if not hasattr(target, "py") or _is_language_specific_proto_library(ctx, target):
+    if not PyInfo in target or _is_language_specific_proto_library(ctx, target):
```

Comment:
This change seems to be breaking for some people...
We get reports of folks getting:
```
ERROR: /private/var/tmp/_bazel_brygidap/0598b1ba445b8f81cd3509e3009d6cb8/external/intellij_aspect/intellij_info_impl.bzl:206:12: name 'PyInfo' is not defined (did you mean 'ApkInfo'?).
```
After this change...